tdrs-backend/tdpservice/parsers/schema_defs/ssp/m3.py

```diff
@@ -0,0 +1,120 @@
+"""Schema for SSP M1 record type."""
+
+
+from ...util import MultiRecordRowSchema, RowSchema, Field
+from ... import validators
+from tdpservice.search_indexes.models.ssp import SSP_M3
+
+first_part_schema = RowSchema(
+    model=SSP_M3,
+    preparsing_validators=[
+        # validators.hasLength(150),  # unreliable.
```

Comment:
As these can be multi-record, this line should be removed given the business case, yes?